### DISCUSSION TITLE
Add more animations

### DISCUSSION THREAD

**Discussion by vollowx** [YaLTeR/niri#202] [Ideas]
1. Switching between workspaces using `focus-window-or-workspace-[up,down]` is animated, but `focus-workspace` isn't
2. Moving windows isn't animated, not necessary though

Hope to see animations here. Nice WM anyway :heart:

EDIT: 1, 2 finished

**Comment by YaLTeR** [comment]
Thanks! I intentionally omitted it because it jumps across multiple workspaces, so it would be a bit jarring. But maybe it's fine. focus-column-first/last is animated at least

**Reply by vollowx** [reply]
Yes it might be harder to implement. As this doesn't really influence daily usage, maybe it could be implemented after the main features.

**Comment by YaLTeR** [comment]
Other window animations are planned, they are just difficult to get right and will need refactoring in the code.

**Comment by tomysurya** [comment]
also if it's possible to add rounded corners would be nice

**Reply by YaLTeR** [reply]
I *just* pushed rounded corners earlier today :)

**Reply by tomysurya** [reply]
great! , how to set up?

**Reply by YaLTeR** [reply]
Here: https://github.com/YaLTeR/niri/wiki/Configuration:-Window-Rules#geometry-corner-radius

**Reply by tomysurya** [reply]
awesome man! thank you!
